viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1224,6 +1227,35 @@ export class Cognite3DViewer {
     return this.intersectModels(offsetX, offsetY);
   }
 
+  /**
+   * Recalcualte the bounding box used to determine near and far planes on the camera. This should only be
+   * necessary if you have custom objects whose bounds change, as this calculation will be done automatically
+   * when models are added to or removed from the viewer.
+   */
+  public forceUpdateCameraNearAndFarBoundingBox(): void {
```

Comment:
Can't this simply update near and far planes, always? Even when automaticNearFarPlane is false. There is no reason anyone would call this if they do not wish to update the near/far planes - is there?